bin/test

```diff
@@ -7,6 +7,12 @@ go get -t ./...
 rm -f coverage.txt
 
 if [ ! -z ${TEST} ]; then
+    if [ ! -z ${TESTRUN} ]; then
+        echo "Running single test $TESTRUN"
+        go test -coverprofile=profile.out -covermode=atomic "$TEST" -test.run "$TESTRUN"
```

Comment:
No known differences. I didn't know about `-run`; it looks like `-run` could be enough by itself since it takes `regexp`.